packages/starlight/components/Icons.ts

```diff
@@ -125,6 +125,8 @@ export const BuiltInIcons = {
 		'<path d="M12.102 0h-.081C5.462 0 .13 5.252.001 11.779v.012L.007 24l12.097-.01c6.582-.055 11.897-5.404 11.897-11.995S18.686.056 12.109 0h-.005zM12 18.857h-.015a6.778 6.778 0 0 1-2.94-.666l.041.018-4.345 1.077 1.227-4.018a6.78 6.78 0 0 1-.83-3.27A6.86 6.86 0 1 1 12 18.857z"/>',
 	zulip:
 		'<path d="M21 19c0 1.7-1.2 3-2.7 3H5.7C4.2 22 3 20.7 3 19a3 3 0 0 1 1.2-2.4l6.7-6c0-.1.3 0 .2.2l-2.5 5s0 .2.2.2h9.5c1.5 0 2.7 1.4 2.7 3Zm0-14a3 3 0 0 1-1.2 2.4l-6.7 6c0 .1-.2 0-.2-.2l2.5-5s0-.2-.2-.2H5.7C4.2 8 3 6.6 3 5c0-1.7 1.2-3 2.7-3h12.6C19.8 2 21 3.3 21 5Z"/>',
+	pinterest:
+		'<path d="M12.02 0a11.99 11.99 0 0 0-4.37 23.15c-.1-.95-.2-2.4.04-3.44l1.4-5.96s-.35-.72-.35-1.78c0-1.66.96-2.9 2.16-2.9 1.03 0 1.52.76 1.52 1.68 0 1.03-.65 2.57-1 4-.28 1.18.6 2.16 1.78 2.16 2.13 0 3.77-2.25 3.77-5.5 0-2.85-2.06-4.86-5-4.86a5.19 5.19 0 0 0-5.41 5.2c0 1.03.39 2.14.88 2.74.1.12.12.22.09.34-.1.38-.3 1.2-.33 1.37-.06.22-.18.27-.4.16-1.5-.69-2.44-2.88-2.44-4.64 0-3.78 2.75-7.26 7.92-7.26 4.16 0 7.4 2.97 7.4 6.93 0 4.13-2.61 7.46-6.24 7.46-1.21 0-2.35-.63-2.76-1.38l-.75 2.85c-.27 1.04-1 2.35-1.5 3.14A12 12 0 1 0 11.99.02l.04-.02z"/>',
```

Comment:
I ran this through [SVGOMG](https://jakearchibald.github.io/svgomg/) and managed to optimize it a little (saves about 70B gzipped):

```suggestion
		'<path d="M12 0a12 12 0 0 0-4.4 23.1c0-.9-.2-2.4 0-3.4l1.5-6s-.4-.7-.4-1.7c0-1.7 1-3 2.2-3 1 0 1.5.8 1.5 1.7 0 1-.6 2.6-1 4-.3 1.2.6 2.2 1.8 2.2 2.1 0 3.8-2.2 3.8-5.5 0-2.8-2-4.9-5-4.9a5.2 5.2 0 0 0-5.4 5.2c0 1 .3 2.2.8 2.8.1.1.2.2.1.3l-.3 1.4c0 .2-.2.3-.4.2-1.5-.7-2.4-3-2.4-4.7C4.4 8 7 4.5 12.3 4.5c4.1 0 7.4 3 7.4 6.9 0 4.1-2.6 7.4-6.3 7.4-1.2 0-2.3-.6-2.7-1.3l-.8 2.8c-.2 1-1 2.4-1.5 3.2A12 12 0 1 0 12 0z"/>',
```

Reply:
Fantastic! I also ran it through SVGOMG but couldn't get it as optimized. Could you share the settings you used?